store/postgres/src/store.rs

```diff
@@ -118,16 +119,20 @@ pub struct StoreConfig {
 }
 
 /// Commonly needed information about a subgraph that we cache in
-/// `Store.subgraph_cache`. Only immutable subgraph data can be cached this
+/// `Store.schema_cache`. Only immutable subgraph data can be cached this
 /// way as the cache lives for the lifetime of the `Store` object
 #[derive(Clone)]
-struct SubgraphInfo {
+struct SchemaCacheEntry {
     /// The schema as supplied by the user
     input: Arc<Schema>,
     /// The schema we derive from `input` with `graphql::schema::api::api_schema`
     api: Arc<Schema>,
     /// The name of the network from which the subgraph is syncing
     network: Option<String>,
+    /// The imported schemas which are referenced by name or were not available to merge
+    unstable_imports: Vec<(SchemaReference, Option<SubgraphDeploymentId>)>,
```

Comment:
If there are unavailable schemas, can the subgraph still function? Can it be queried?